interact.py

```diff
@@ -89,26 +92,35 @@ def do_add_person(self, arg):
 
     @docopt_cmd
     def do_print_room(self, arg):
-        """Usage: print_room (office | living) <room_name>... """
+        """Usage: print_room <room_name>... """
 
         rooms = arg['<room_name>']
-        if arg['office']:
-            is_office = True
-        else:
-            is_office = False
-
-        result = dojo.print_room(is_office, rooms)
-        null = result[0]
-        existing = result[1]
-
-        if result[2]:
-            for room in existing:
-                print(room, ":=>", dojo.offices[room])
-        else:
-            for room in existing:
-                print(room, ":=>", dojo.livings[room])
-
+        result = dojo.print_room(rooms)
+
+        offices = result[0]
```

Comment:
I think this logic should be done on your class file and not the front end. Print to screen from your class not from the docopt interface to avoid logic at the front